tests/App/config.yml

```diff
@@ -18,3 +18,8 @@ services:
     Sonata\BlockBundle\Tests\App\Controller\DemoController:
         tags:
             - controller.service_arguments
+
+sonata_block:
+    http_cache:
+        handler: sonata.block.cache.handler.noop
+        listener: false
```

Comment:
It's kinda sad to ask people adding some extra options to avoid deprecation, and then removing the option in next major. I dunno if we have another solution...

Do we have another solution @sonata-project/contributors ?

Reply:
Currently the http cache is enabled by default. So to have the same behavior as on 5.0 people should somehow disable it I think. This would have lead to less surprises when upgrading to 5.0.

Maybe someone has a better idea? We could also just allow disabling it by setting `http_cache: false` maybe. At least thats simpler.

Reply:
We would deprecate it on 5.x then and remove on 6.0